geode-wan/src/test/java/org/apache/geode/internal/cache/wan/parallel/ParallelGatewaySenderOperationsDUnitTest.java

```diff
@@ -195,10 +196,11 @@ public void testParallelPropagationSenderResume() throws Exception {
     vm6.invoke(() -> resumeSender("ln"));
     vm7.invoke(() -> resumeSender("ln"));
 
+    async.join(120000);
```

Comment:
Jason, I improved the API in AsyncInvocation, so I recommend using this syntax (the 1st one has a default timeout of 30 seconds):
```
async.await();
```
or
```
async.await(2, TimeUnit.MINUTES);
```

Reply:
Oh cool, thanks!